drivers/usb/device/Kconfig

```diff
@@ -82,6 +82,14 @@ config USB_KINETIS
 	help
 	  Kinetis USB Device Controller Driver.
 
+config USB_DC_NXP_EHCI
+	bool "Kinetis and RT EHCI USB Device Controller Driver"
+	depends on HAS_MCUX_USB_EHCI
+	select USB_DEVICE_DRIVER
+	select NOCACHE_MEMORY if HAS_MCUX_CACHE
```

Comment:
Please update the commit message to reflect that you're no longer adding `select ARM_MPU if CPU_CORTEX_M7`.